src/App.js

```diff
@@ -1,13 +1,11 @@
+import React from 'react';
 import './App.css';
+import Calc from './Components/calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <h1>Math Magician</h1>
-      </header>
-    </div>
-  );
-}
+const App = () => (
```

Comment:
- [ ] This should be a class component instead of a functional one, kindly make this change 😄